R/translate-sql-conditional.R

```diff
@@ -66,5 +81,18 @@ sql_switch <- function(x, ...) {
 }
 
 sql_is_null <- function(x) {
-  sql_expr((((!!x)) %is% NULL))
+  x_expr <- enexpr(x)
+  if (is_call(x_expr)) {
```

Comment:
Can you now use `enpar` here?

Reply:
Yes, and thanks to that I found an issue with this approach and the pipe